### PR TITLE
added statistics comp

### PR DESCRIPTION
**Pull request by meghanavaitla**
[
statistics
](url)

### CODE REVIEW

File: src/components/statistics.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        paddingLeft: '100px'
+    }
+
+}))
+
+export default function Statistics() {
+
+    const classes = useStyles();
+    return (
+        <Grid container spacing={3} className={classes.grid}>
```
Comment (keerthi-reddy) [inline src/components/statistics.js]: Making the grid dynamic would be better i guess!!

File: src/components/statistics.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            <Grid item xs={12} lg={3} md={3} sm={6}>
+                <Paper className={classes.paper}>
+                    <h1><CountUp end={72} className={classes.val} /></h1>
```
Comment (keerthi-reddy) [inline src/components/statistics.js]: Use typography instead of h1 and p

File: src/components/statistics.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import React from "react"
+import { makeStyles } from '@material-ui/core/styles';
+import { Grid, Paper } from '@material-ui/core';
+import CountUp from 'react-countup';
+const useStyles = makeStyles((theme) => ({
```
Comment (keerthi-reddy) [inline src/components/statistics.js]: Maintain styles in seperate file

File: src/components/statistics.js (hunk 1 above)
Comment (meghanavaitla) [reply src/components/statistics.js]: ok, i will try that

File: src/components/statistics.js (hunk 2 above)
Comment (meghanavaitla) [reply src/components/statistics.js]: okay

File: src/components/statistics.js (hunk 3 above)
Comment (meghanavaitla) [reply src/components/statistics.js]: okay

File: src/assets/stats.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        {
+            "name": "Hosted Contests",
+            "value": "15"
+        },
+        {
+            "name": "Sessions Taken",
+            "value": "48"
+        }
+    ]
+}
```
Comment (pinnamanivenkat) [inline src/assets/stats.json]: please add a new line to the file.

File: src/assets/stats.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{
+    "statistics": [
```
Comment (pinnamanivenkat) [inline src/assets/stats.json]: need to have "statistics" as a key
```json
[
  {},
  {}
]
```

File: src/components/statistics-style.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import { makeStyles } from '@material-ui/core/styles';
+const useStyles = makeStyles((theme) => ({
+    paper: {
+        padding: theme.spacing(3),
+        textAlign: 'center'
+    },
+
+    name: {
+        fontFamily: "RaleWay",
+        marginTop: '10px'
```
Comment (pinnamanivenkat) [inline src/components/statistics-style.js]: make use of vh instead of px

File: src/assets/stats.json [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+        {
+            "name": "Club Members",
+            "value": "72"
+        },
+        {
+            "name": "Participations",
+            "value": "7"
+        },
+        {
```
Comment (pinnamanivenkat) [inline src/assets/stats.json]: perhaps add an image as well?
Makes it look good. 
https://codepen.io/syedrafeeq/pen/rcfsJ

**Review by pinnamanivenkat — CHANGES_REQUESTED** [review]

**Comment by pinnamanivenkat** [comment]
Loved it!

**Review by pinnamanivenkat — APPROVED** [review]

**Pull request merged**